packages/client/src/views/SysAdmin/Config/Certificates/Certificates.tsx

```diff
@@ -332,10 +332,13 @@ class CertificatesConfigComponent extends React.Component<Props, State> {
         event
       )
       if (res && res.createOrUpdateCertificateSVG) {
+        // Sometimes it takes a second for the Minio URL to be available, otherwise you get a 404.
         this.setState({ imageUploading: false })
-        this.props.updateOfflineCertificate(
-          res.createOrUpdateCertificateSVG as ICertificateTemplateData
-        )
+        setTimeout(() => {
+          this.props.updateOfflineCertificate(
+            res.createOrUpdateCertificateSVG as ICertificateTemplateData
+          )
+        }, 1000)
```

Comment:
This also fixes a bug when uploading a certificate template. The URL isnt retrievable immediately from Minio causing a 404